Algorithms/Sorters/External/Storages/IntFileStorage.cs

```diff
@@ -0,0 +1,43 @@
+using System.IO;
+
+namespace Algorithms.Sorters.External.Storages
+{
+    public class IntFileStorage : ISequentialStorage<int>
+    {
+        private readonly string filename;
+
+        public IntFileStorage(string filename, int length)
+        {
+            Length = length;
+            this.filename = filename;
+        }
+
+        public int Length { get; }
+
+        public ISequentialStorageReader<int> GetReader() => new FileReader(filename);
+
+        public ISequentialStorageWriter<int> GetWriter() => new FileWriter(filename);
+
+        private class FileReader : ISequentialStorageReader<int>
```

Comment:
Codacy found an issue: [Implement 'IDisposable' in this class and use the 'Dispose' method to call 'Dispose' on 'reader'.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6254880)